hub2vu/rag_chatbot.ipynb

```
@@ -0,0 +1,252 @@
+    "\n",
+    "    당신은 전문적인 지식을 갖춘 듯 정확하고, 동시에 친절하고 이해하기 쉬운 어투를 구사합니다. \n",
+    "    \"\"\"\n",
+    "\n",
+    "    user_prompt = f\"\"\"아래는 검색된 문서들의 내용입니다:\n",
+    "    {context_str}\n",
+    "    질문: {query}\"\"\"\n",
+    "\n",
+    "    # ChatGPT 호출\n",
+    "    api_key = os.getenv(\"OPENAI_API_KEY\")\n",
+    "    client = OpenAI(api_key=api_key)\n",
+    "\n",
```

Ruff (0.13.1)

`from tkinter import *` used; unable to detect undefined names

(F403)